lib/sandboxed_renderer/init.js

```diff
@@ -73,6 +77,9 @@ function preloadRequire (module) {
 // and any `require('electron')` calls in `preload.js` will work as expected
 // since browserify won't try to include `electron` in the bundle, falling back
 // to the `preloadRequire` function above.
+if (!preloadPath)
+  return
```

Comment:
i don't think you can return outside the context of a function

Reply:
The file is wrapped in an anonymous function in order to be executed, though this should break linting